helper-cli/src/main/kotlin/commands/GeneratePackageConfigurationsCommand.kt

```diff
@@ -77,11 +85,21 @@ internal class GeneratePackageConfigurationsCommand : CliktCommand(
 
     private fun Provenance.writePackageConfigurationFile(filename: String) {
         val packageConfiguration = createPackageConfiguration(packageId, this)
-        val outputFile = outputDir.resolve(filename)
+        val outputFile = getOutputFile(filename)
 
         packageConfiguration.writeAsYaml(outputFile)
         println("Wrote a package configuration to '${outputFile.absolutePath}'.")
     }
+
+    private fun getOutputFile(filename: String): File {
+        val relativeOutputFilePath = if (createHierarchicalDirs) {
+            "${packageId.toPath(emptyValue = "_")}/$filename"
```

Comment:
Now my next obvious question is: *Why* to we use `_` here but `unknown` in ORT core? Wouldn't it make sense to align the two? I actually like `_` better than `unknown`, I have to admit, but I believe there was a reason why we did not go for `_` in ORT core... but I cannot recall it right now.

Reply:
Yes, I'm fine with keeping `_` here (and `unknown` in core), I just wanted to get discussion going.

Reply:
Do you think adding a command line options as mentioned above is a good idea?

Reply:
No, I think that would over-do it.

BTW, I just created https://github.com/oss-review-toolkit/ort/issues/2987.